plugins/modules/nd_setup.py

```diff
@@ -0,0 +1,471 @@
+    description:
+    - The service subnet.
+    - This is a virtual level subnet used for communication within the cluster.
+    type: str
+  app_network_ipv6:
+    description:
+    - The IPv6 app subnet.
+    - This option is only applicable for ND version 3.0.1f and its variants.
+    type: str
+  service_network_ipv6:
+    description:
+    - The IPv6 service subnet.
+    - This option is only applicable for ND version 3.0.1f and its variants.
+    type: str
+  ntp_config:
+    description:
+    - This is used for configuring NTP server and its other options.
+    - This option is only applicable and required for ND version 3.0.1f and its variants.
+    type: dict
+    suboptions:
+      servers:
+        description:
+        - This option is used for setting up the NTP host.
+        type: list
+        required: true
+        elements: dict
+        suboptions:
+          ntp_host:
+            description:
+            - The IP address of the NTP server.
+            type: str
+            required: true
+          ntp_key_id:
+            description:
+            - The NTP Key ID.
+            type: int
+          preferred:
+            description:
+            - The preferred status of the NTP host.
+            type: bool
+            default: false
+      keys:
+        description:
+        - This option is used for setting up the NTP keys.
+        type: list
+        elements: dict
+        suboptions:
+          ntp_key_id:
+            description:
+            - The NTP Key ID.
+            type: int
+            required: true
+          ntp_key:
+            description:
+            - The value of the NTP key.
+            type: str
+            required: true
+          authentication_type:
+            description:
+            - The authentication type for the NTP key.
+            type: str
+            choices: [ AES128CMAC, SHA1, MD5 ]
+            required: true
+          trusted:
+            description:
+            - The trusted status of the NTP key.
+            type: bool
+            default: false
+  nodes:
+    description:
+    - The node details to set up Nexus Dashboard and bring up the User Interface.
+    type: list
+    elements: dict
+    suboptions:
+      hostname:
+        description:
+        - The host name of the node.
+        type: str
+        required: true
+      serial_number:
+        description:
+        - The serial number of the node.
+        type: str
+        required: true
+      management_ip_address:
+        description:
+        - The management IP address of the node.
+        type: str
+        required: true
+      username:
+        description:
+        - The username of the node.
+        type: str
+        required: true
+      password:
+        description:
+        - The password of the node.
+        type: str
+        required: true
+      management_network:
+        description:
+        - The network that is used for DNS and NTP communication.
```

Comment:
- The network that is used for DNS and NTP communication. -> - The network used for DNS and NTP communication.

Reply:
ok